airflow/utils/metastore_cleanup.py

```diff
@@ -0,0 +1,310 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
https://github.com/apache/airflow/pull/20838/files#r807326376 will affect the module name. how about `airflow/utils/db_cleanup.py`